commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -278,6 +280,7 @@ class Service(_AnsiblePlaybookAlias):
         'pg_standby': ['postgresql', 'pgbouncer'],
         'celery': ['celery'],
         'webworkers': ['webworkers'],
+        'formplayer': ['formplayer', 'formplayer-spring']
```

Comment:
I would actually push for these two to be defined separately like this:

```
'formplayer': ['formplayer-spring'],
'touchforms': ['formplayer'],
```

Why? These supervisor services are kind of poorly named as a historical accident. What we now call "formplayer" is backed by the supervisor service named `formplayer-spring`, because what we used before, touchforms, was already being run under the name `formplayer`.

Reply:
oh thats quite confusing.. thanks for clearing that out. Will update.